src/graph_notebook/magics/graph_magic.py

```diff
@@ -349,8 +349,9 @@ def gremlin(self, line, cell, local_ns: dict = None):
 
             try:
                 logger.debug(f'groupby: {args.group_by}')
+                logger.debug(f'ignore_groups: {args.ignore_groups}')
                 if args.ignore_groups:
-                    gn = GremlinNetwork()
+                    gn = GremlinNetwork(ignore_groups=args.ignore_groups)
```

Comment:
We should be able to specify a default value for `args.ignore_groups` and then just provide it in all cases